src/routes/v2/authenticated/__tests__/review.spec.ts

```diff
@@ -136,6 +136,10 @@ describe("Review Requests Router", () => {
   })
 
   describe("compareDiff", () => {
+    beforeEach(() => {
+      // NOTE: Skip preconditions check for unmigrated site
```

Comment:
Can we put as a TODO to remove this after email migration is complete?

Reply:
sure, changing to `TODO (IS-58)` inline with the actual code itself